mycroft/messagebus/send.py

```diff
@@ -14,29 +14,30 @@
 #
 import sys
 import json
-from mycroft.messagebus.client.ws import WebsocketClient
-from mycroft.messagebus.message import Message
-from mycroft.configuration import ConfigurationManager
+
 from websocket import create_connection
 
+from mycroft.configuration import ConfigurationManager
+from mycroft.messagebus.client import MessageBusClient
+from mycroft.messagebus.message import Message
+
 
 def main():
-    """
-        Main function, will run if executed from command line.
+    """Main function, will run if executed from command line.
```

Comment:
Space after `"""`

Reply:
I was following PEP257 when making docstring changes:https://www.python.org/dev/peps/pep-0257/

Reply:
Can't argue with the PEP's even if it's like sandpaper on my eye :P